src/utils.ts

```diff
@@ -72,6 +72,50 @@ const hexToHexAlpha = (hex: string, alpha: number): string => {
 	return `#${hex}${alphaHex}`;
 };
 
+export async function fetchGitHubContributions(username: string): Promise<number[][]> {
```

Comment:
We already have a function called `getGithubContribution`, Why do we need the new function?

Also `https://github.com/users/${username}/contributions` endpoint limits the number of fetched contributions.